src/virtualenv/create/via_global_ref/builtin/python2/site.py

```diff
@@ -6,15 +6,29 @@
 """
 import sys
 
+base_executable = None
```

Comment:
please don't, don't introduce globals they just make code hard to follow

Reply:
Ok, though the list of arguments is going to be long to carry everwhere :(

Reply:
I personally prefer that over globals. Also, we don't have to carry everything everywhere.